bin/generate-apidocs.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+const spawn = require('child_process').spawn;
+// Call sdocs
+spawn(
+  process.execPath,
+  ['../../node_modules/strong-docs/bin/cli.js', '-o', 'api-docs']
```

Comment:
Please consider using `require.resolve('strong-docs/bin/cli')`, as it is more robust and will handle situations where strong-docs may happen to be installed higher up in the filesystem (e.g. `../../../../node_modules/strong-docs/bin/cli.js`).